bwa/0.7.17/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial
+FROM ubuntu:xenial as app
 
 LABEL base.image="ubuntu:xenial"
```

Comment:
Looks like these labels are from the prior dockerfile, and one of the now-required ones is missing:

`dockerfile.version`

Could you add the following?
```
LABEL dockerfile.version="2"
```